client/components/chart/bar.jsx

```diff
@@ -107,11 +107,14 @@ module.exports = React.createClass( {
 		};
 
 		if ( this.props.data.tooltipData && this.props.data.tooltipData.length && ! this.props.isTouch ) {
-			tooltip = <Popover context={ this.refs && this.refs.valueBar }
+			tooltip = <Popover
+							id="popover__chart-bar"
```

Comment:
This `id` is optional not needed. For now I'm using this `id` JUST for debugging.

![image](https://cloud.githubusercontent.com/assets/77539/17374816/b7c9331c-5984-11e6-9137-97fdd8b1bf82.png)

![image](https://cloud.githubusercontent.com/assets/77539/17374848/e1c4d6bc-5984-11e6-9518-6ffa31382284.png)

it's a mess but for now we don't have an isolation environment to check.

Also it could be useful to try to avoid creating multiple popovers when you could use only one for a bunch of targets but I haven't done anything to trying to tackle this scenario.